@xen-orchestra/web-core/lib/components/ui/alert/UiAlert.vue

```diff
@@ -0,0 +1,93 @@
+<!-- v3 -->
+<template>
+  <div :class="toVariants({ accent })" class="ui-alert">
+    <div class="content">
+      <VtsIcon class="information-icon" :accent :icon="faCircle" :overlay-icon="icon" />
+      <div class="alert">
+        <div class="typo-body-regular-small">
+          <slot />
+        </div>
+        <div v-if="slots.description" class="typo-body-regular-small">
+          <slot name="description" />
+        </div>
+      </div>
```

Comment:
I think we can refactor a little:
```suggestion
      <div class="alert typo-body-regular-small">
        <div>
          <slot />
        </div>
        <div v-if="slots.description">
          <slot name="description" />
        </div>
      </div>
```